src/app/zap-templates/templates/app/CHIPClusters-src.zapt

```diff
@@ -55,8 +55,8 @@ namespace {
   // Pick source endpoint as 1 for now
   constexpr EndpointId kSourceEndpoint = 1;
 
-  const uint8_t kReportingDirectionReported = 0x00;
-}
+  [[maybe_unused]] const uint8_t kReportingDirectionReported = 0x00;
```

Comment:
Can we 'constexpr' this instead? do we need to allocate memory here? maybe_unused is suspicious

Reply:
Realistically these should go in a header. "generated code"  should always rely on generated data not be hardcoded.

Reply:
> Can we 'constexpr' this instead? do we need to allocate memory here? maybe_unused is suspicious
I would only be able to address this next week.

Reply:
Opened https://github.com/project-chip/connectedhomeip/issues/9189 to track this.